world.cpp

```diff
@@ -42,7 +42,12 @@ LogMapNumberResult_t load_map_number(S32 map_number, Coord_t* player_start, Worl
 
      // search through directory to find file starting with 3 digit map number
      DIR* d = opendir("content");
-     if(!d) return result;
+     if(!d){
+         printf("load_map_number: opendir() failed\n");
```

Comment:
To get a more helpful error message (and use the LOG) on why opendir fails, can we try:
```
LOG("load_map_number(): opendir() failed: %s\n", strerror(errno));
```

According to the man page for opendir http://man7.org/linux/man-pages/man3/opendir.3.html